src/covid_model_seiir_pipeline/pipeline/forecasting/specification.py

```diff
@@ -78,51 +75,32 @@ class ScenarioSpecification:
         'offset_deaths_lower',
         'offset_deaths_upper',
     )
-    VARIANT_KEYS = (
-        'version',
-        'beta_scalar',
-        'ifr_scalar',
-    )
 
     name: str = field(default='dummy_scenario')
     algorithm: str = field(default='normal')
     algorithm_params: Dict = field(default_factory=dict)
-    solver: str = field(default='RK45')
     system: str = field(default='normal')
-    system_params: Dict = field(default_factory=dict)
-    population_partition: str = field(default='none')
     beta_scaling: Dict[str, int] = field(default_factory=dict)
-    variant: Dict[str, Any] = field(default_factory=dict)
     theta: Union[str, int] = field(default=0)
+    vaccine_version: str = field(default='reference')
+    variant_ifr_scale: float = field(default=1.29)
+    probability_cross_immune: float = field(default=0.0)
     covariates: Dict[str, str] = field(default_factory=dict)
 
     def __post_init__(self):
         if self.algorithm not in self.ALLOWED_ALGORITHMS:
             raise ValueError(f'Unknown algorithm {self.algorithm} in scenario {self.name}. '
                              f'Allowed algorithms are {self.ALLOWED_ALGORITHMS}.')
 
-        if self.solver not in self.ALLOWED_SOLVERS:
-            raise ValueError(f'Unknown solver {self.solver} in scenario {self.name}. '
-                             f'Allowed solvers are {self.ALLOWED_SOLVERS}.')
-
         if self.system not in self.ALLOWED_SYSTEMS:
             raise ValueError(f'Unknown system {self.system} in scenario {self.name}. '
                              f'Allowed systems are {self.ALLOWED_SYSTEMS}.')
 
-        if self.population_partition not in self.ALLOWED_POPULATION_PARTITIONS:
-            raise ValueError(f'Unknown population partition {self.population_partition} in scenario {self.name}. '
-                             f'Allowed partitions are {self.ALLOWED_POPULATION_PARTITIONS}.')
-
         bad_scaling_keys = set(self.beta_scaling).difference(self.BETA_SCALING_KEYS)
         if bad_scaling_keys:
             raise ValueError(f'Unknown beta scaling configuration option(s) {list(bad_scaling_keys)} '
                              f'in scenario {self.name}. Expected options: {self.BETA_SCALING_KEYS}.')
 
-        bad_variant_keys = set(self.variant).difference(self.VARIANT_KEYS)
-        if bad_variant_keys:
-            raise ValueError(f'Unknown variate configuration options(s) {list(bad_variant_keys)} '
-                             f'in scenario {self.name}. Expected options: {self.VARIANT_KEYS}.')
-
```

Comment:
why aren't any of these guards necessary now?

Reply:
The configuration keys they were guarding against got deleted.